app/views/ems_physical_infra/_main.html.haml

```diff
@@ -0,0 +1,11 @@
+= render :partial => "layouts/flash_msg"
+.row
+  .col-md-12.col-lg-6
```

Comment:
This is obsolete and doesn't work anymore. Please see https://github.com/ManageIQ/manageiq-ui-classic/pull/274 on how to fix this.

Also, @juliancheal .. clearly this screen is not being tested since it doesn't work and the tests pass ;).